### ISSUE TITLE
Submit to devdocs?

### ISSUE THREAD

**Issue by chmac** [remeda/remeda#263]
Firstly, thanks a lot to all the folks who work on remeda. It's awesome to have properly typed utility functions. 🧡 🧡 🧡 

My goto reference for documentation is devdocs.io. Would it make sense to seek adding remeda to devdocs?

**Comment by TkDodo** [comment]
didn't know that site, but yes, if you could contribute adding remeda to it, that would be great ❤️

**Comment by chmac** [comment]
@TkDodo I think they use a format which is also compatible with a famous desktop docs program for mac (but I forget the name of it right now!).

It likely involves a build process being run to generate the docs in the required format. Then that output needs to be published in some manner. Do you think there's a willingness to add that to the remeda build process and maintain it?

**Comment by TkDodo** [comment]
if you'd like to contribute it and it's automated, sure. I won't have time in the near future to do this myself

**Comment by chmac** [comment]
Awesome, that's great. I'm having a quick look at the process now. I might have misunderstood, it's possible that it only requires work on the devdocs site. But if I don't figure it out or find the time, we could leave this issue open as a nice "good first issue" candidate for folks who might want to contribute in the future.

**Comment by uinz** [comment]
> @TkDodo I think they use a format which is also compatible with a famous desktop docs program for mac (but I forget the name of it right now!).
> 
> It likely involves a build process being run to generate the docs in the required format. Then that output needs to be published in some manner. Do you think there's a willingness to add that to the remeda build process and maintain it?

I guess you means `Dash`

**Comment by chmac** [comment]
@uinz Yes, that's the one! Then there's an open source version which is for non-mac platforms I believe. But now that I looked into it a bit, it seems like DevDocs uses their own crawler, so I think I was mistaken about the formats being interoperable.

There's some info on how to suggest docs here:
https://trello.com/c/PVnfdeaN/26-suggest-new-docs-here

They require >5k GitHub stars. I think we could still submit a PR to add remeda, but I think that the project maintainers won't work on anything which has less than 5k stars. I'm not 100% sure about that though, perhaps remeda is not eligible at all until it reaches 5k stars. Guess it's not too far away, we're at 2.6k today... :-)

**Comment by cjquines** [comment]
i don't think we're anywhere near the popularity needed to get into devdocs :sweat_smile:

**Comment by chmac** [comment]
@cjquines It was 2.6k stars ~14 months ago and today it's 4.2k, in theory with linear growth that would mean 5k in ~7 months. That seems pretty close to me, but it seems you think this is not worth pursuing?

**Comment by eranhirsch** [comment]
Honestly, looking at the project, it feels like if you push a PR to add the docs, they won't reject it just because of the star rating (but I might be wrong).

**Comment by cjquines** [comment]
fair enough. i'll reopen
